app/controllers/events/email.py

```diff
@@ -0,0 +1,53 @@
+from flask import Flask, redirect, flash, url_for, request, g, render_template
+from flask_mail import Mail, Message
+from peewee import fn
+from app.models.interest import Interest
+from app.models.user import User
+from app.models.event import Event
+from app.models.eventParticipant import EventParticipant
+from app.models.programEvent import ProgramEvent
+from app.models.emailTemplate import EmailTemplate
+from app.models.emailLog import EmailLog
+from app.controllers.main import main_bp
+from app.logic.emailHandler import getInterestedEmails, getParticipantEmails, emailHandler
+from app import app
+
+@main_bp.route('/retrieveEmailTemplate', methods=['GET'])
+def retrieveEmailTemplate():
+    # Couldn't find a way to do this with render_template

```

Comment:
If you're not sure how to set it up on the javascript side so the ajax populates the modal from the response, you could use `render_template` here and store the resulting html under the `'body'` key, rather than storing the template.